pyiron_contrib/workflow/channels.py

```diff
@@ -199,9 +212,13 @@ def ready(self) -> bool:
             (bool): Whether the value matches the type hint.
         """
         if self.type_hint is not None:
-            return valid_value(self.value, self.type_hint)
+            return self.value_is_data and valid_value(self.value, self.type_hint)
         else:
-            return True
+            return self.value_is_data
+
+    @property
+    def value_is_data(self):
```

Comment:
Does it make maybe sense to make it private?

Reply:
Yeah, just to avoid cluttering the tab completion list, sure.

I'm going to go have some breakfast, then I'll get this change in.